README.md

```diff
@@ -30,3 +30,50 @@ $ gulp
 Then point your browser to [http://localhost:8000](http://localhost:8000/).
 
 Note: The site is built in the `dist/app` folder.
+
+## Building for Cordova
+
+Run `npm install -g cordova` to install Cordova, and make sure you have the Android
+emulator installed. Then, to build this app as a Cordova app, run:
+
+```bash
+gulp build
+cd dist
+cordova create cordova
+rm -r cordova/www/*
+cp -r app/* cordova/www/
+rm cordova/www/cordova.js
+cd cordova
+cordova plugin add https://github.com/michielbdejong/SecureHTTP.git
+cordova platform add android
+cordova build android
+cordova emulate android
+```
```

Comment:
Done. I couldn't merge `gulp cordova-setup` and `gulp cordova-android` into one task because the second has to run in a folder (`dist/cordova`), which the first one creates. So I left it as two tasks, hope that's ok.